ephios/core/views/settings.py

```diff
@@ -21,6 +21,13 @@ def get_available_management_settings_sections(request):
                 "active": request.resolver_match.url_name == "settings_instance",
             }
         )
+        sections.append(
+            {
+                "label": _("OAuth2 applications"),
+                "url": reverse("oauth2_provider:list"),
+                "active": request.resolver_match.url_name == "list",
```

Comment:
I have the feeling we must be more specific with the `active` predicate here.

Reply:
do you have an idea how? I didn't wanna change the url_names of oauth2_provider

Reply:
Resolver_match should also provided the app name via an attribute?!